go-controller/pkg/ovn/gateway_init.go

```diff
@@ -371,6 +371,7 @@ func (oc *Controller) gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet,
 					DoAfter: func() {
 						logicalRouter.StaticRoutes = libovsdbops.ExtractUUIDsFromModels(&logicalRouterStaticRouteRes)
 					},
+					BulkOp: true,
```

Comment:
@tssurya just a quick note here. Since the predicate could potentially return more than one row, it is a good idea to have `BulkOp: true` added.

Reply:
oh cool nice thanks! I'll keep this in mind for future